### ISSUE TITLE
Allow pid or host:port entry in the process picker

### ISSUE THREAD

**Issue by tmm1** [microsoft/vscode-js-debug#2078]
**Is your feature request related to a problem? Please describe.**

The regexp for showing available processes can be quite frustrating, esp when I already know the target pid/port I'm trying to debug.

**Describe the feature you'd like**

Under `Debug: Attach to Node Process` when a dialog appears with a text box placeholder `Pick the node.js process to attach to`, I would love to be able to paste something like 1234 or 127.0.0.1:56777

**Comment by connor4312** [comment]
This works for me, e.g. here's filtering for a PID 21172 and also a node process listening on 21172. Could you share more about the scenario you expect to work?

![image](https://github.com/user-attachments/assets/0fedae6b-d8f3-4215-b61d-108b09500a9f)

**Comment by tmm1** [comment]
I want to attach to vscode insiders exthost

Thus the process name does not say node, and it won't appear.

This kind of situation is more common now with features like https://nodejs.org/api/single-executable-applications.html

**Comment by tmm1** [comment]
It looks like the picker won't accept inputs easily.. the input is just used as a filter over the picker choices?

https://github.com/microsoft/vscode-js-debug/blob/9419f572e768563273aca4e28179d8d5413cf5e0/src/ui/processPicker.ts#L135-L139

Maybe it's simpler to tackle this with a user-override on the regex instead, ala https://github.com/microsoft/vscode-js-debug/issues/1480#issuecomment-1353061341

**Comment by connor4312** [comment]
Or maybe just an option to "Show all processes..." in the picker

**Comment by tmm1** [comment]
You're right, an escape hatch to be able to filter over all the processes with a pid/port or name like `(Plugin)` would work splendidly.
